src/WebKit/Enumerators.cs

```diff
@@ -24,23 +26,25 @@ internal class IndexedContainerEnumerator<T> : IEnumerator<T> {
 
 		public T Current {
 			get {
-				return _container [_index];
+				return _container! [_index];
 			}
```

Comment:
Highlighting this `!`. The return type cannot be made nullable since it is implementing an interface with that method signature but it is also a getter that is not an indexer so I thought this may be the best option

Reply:
what about using:
```csharp 
if (_container is null) {
    return default (T);
} else {
    return _container [_index]
}
```

Reply:
@rolfbjarne  good one, that exception should also be a good idea for my other comment.